source/docs/api-reference/api-usage/talonfx-control-requests/closed-loop-requests.rst

```diff
@@ -27,6 +27,7 @@ Velocity closed loop is currently supported for all base :ref:`control output ty
 
 In a Velocity closed loop, the gains should be configured as follows:
 
+- :math:`K_s` - output to overcome static friction (output)
 - :math:`K_v` - output per unit of requested velocity (output/rps)
 - :math:`K_p` - output per unit of error in velocity (output/rps)
 - :math:`K_i` - output per unit of integrated error in velocity (output/rotation)
```

Comment:
I think the order of gains should go P -> I -> D -> V -> S

Reply:
I disagree. Feedforward terms are typically kS, kV, and kA (as seen in WPILib), so the order should either be:
- S -> V -> P -> I -> D (current)
- P -> I -> D -> S -> V

I personally prefer putting the feedforward terms first, as feedforward should do most of the work, and not all PID terms are necessary for a given mechanism.

Reply:
Gotcha. That's fine. Feedforward first should be the first approach to closed-loop controls anyways.